bindep.txt

```diff
@@ -2,3 +2,4 @@
 # see https://docs.openstack.org/infra/bindep/ for additional information.
 
 gcc [compile platform:centos-8 platform:rhel-8]
+gcc [compile platform:rhel-9]
```

Comment:
I belive you can just keep everything in one single line:

`gcc [compile platform:centos-8 platform:rhel-8 platform:rhel-9]`

Reply:
I guess that could work too, tried to follow example given in the Jira task
https://github.com/ansible/awx/pull/11950/files